docs/usage/webworker.md

```diff
@@ -61,72 +55,43 @@ const context = {
 };
 
 async function main() {
-  try {
-    const { results, error } = await asyncRun(script, context);
-    if (results) {
-      console.log("pyodideWorker return results: ", results);
-    } else if (error) {
-      console.log("pyodideWorker error: ", error);
-    }
-  } catch (e) {
-    console.log(
-      `Error in pyodideWorker at ${e.filename}, Line: ${e.lineno}, ${e.message}`,
-    );
+  const { result, error } = await asyncRun(script, context);
+  if (result) {
+    console.log("pyodideWorker result:", result);
+  } else if (error) {
+    console.log("pyodideWorker error:", error);
   }
 }
 
 main();
 ```
 
-Before writing the API, let's first have a look at how the worker operates.
-How does our web worker run the `script` using a given `context`.
+Before writing the API, let's first have a look at how the worker runs the
+`script` using a given `context`.
 
 ### Web worker
 
-Let's start with the definition. [A worker][worker api] is:
-
-> A worker is an object created using a constructor (e.g. [Worker()][worker constructor]) that runs a named JavaScript file — this file contains the code that will run in the worker thread; workers run in another global context that is different from the current window. This context is represented by either a DedicatedWorkerGlobalScope object (in the case of dedicated workers - workers that are utilized by a single script), or a SharedWorkerGlobalScope (in the case of shared workers - workers that are shared between multiple scripts).
-
-In our case we will use a single worker to execute Python code without interfering with
-client side rendering (which is done by the main JavaScript thread). The worker does
-two things:
-
-1. Listen on new messages from the main thread
-2. Respond back once it finished executing the Python script
-
-These are the required tasks it should fulfill, but it can do other things. For
-example, to always load packages `numpy` and `pytz`, you would insert the line
-{js:func}`await pyodide.loadPackage(['numpy', 'pytz']); <pyodide.loadPackage>` as
-shown below:
+Here is the worker code:
 
 ```js
-// webworker.js
+// webworker.mjs
+import { loadPyodide } from "https://cdn.jsdelivr.net/pyodide/v0.26.2/full/pyodide.mjs";
```

Comment:
Oops, yes. It would be nice to have a more systematic way to test these. If we put the three files in a separate folder and then used some sort of include directive to get them into the docs, that would help a lot. But then we'd also need some way to deal with `PYODIDE_CDN_URL`.